ci/debian/control

```diff
@@ -187,9 +187,11 @@ Depends:
     centreon-gorgone (>= ${centreon:version}~),
     centreon-engine-extcommands (>= ${centreon:version}~),
     centreon-trap (>= ${centreon:version}~),
+    centreon-plugin-applications-databases-mysql,
     centreon-plugin-applications-monitoring-centreon-central,
+    centreon-plugin-applications-monitoring-centreon-database,
+    centreon-plugin-applications-monitoring-centreon-map4-jmx,
```

Comment:
@lgcosta Please consider also adding `centreon-plugin-applications-jvm-actuator` as the latest version of Centreon Map cannot be monitored with JMX/Mbeans.

Reply:
added, thanks @Sims24